src/NativeUi.elm

```diff
@@ -111,19 +111,14 @@ unsafeRenderDecodedProperty =
 
 {-| -}
 style : List Style.Style -> Property msg
-style styles =
-    Style.encode styles
-        |> Native.NativeUi.style
+style =
+    Native.NativeUi.style << Style.encode
 
 
 {-| -}
 on : String -> Decoder msg -> Property msg
-on eventName =
-    let
-        realEventName =
-            "on" ++ eventName
-    in
-        Native.NativeUi.on realEventName
+on =
+    Native.NativeUi.on << ((++) "on")
```

Comment:
Not 100% if this was a great idea since it sort of disguises the intent of prepending `"on"`. Maybe it'd be an idea to write a `getEventName` function that is either just `((++) "on")` or something "safer" like
```elm
getEventName eventName =
    if String.startsWith "on" eventName then
        eventName
    else
        "on" ++ eventName
```